.github/workflows/check_format.yml

```diff
@@ -15,7 +15,7 @@ on:
 
 jobs:
   check:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
Please specify the version (I suppose latest means `22.04` now).
Otherwise we might unexpectedly end up with incompatible versions.

Reply:
Ok, done. Are you ok with the `check_license_header.yaml` workflow uses `ubuntu-latest`? It doesn't install anything so there's a smaller chance of surprises there, but still.